extensions/amp-story-audio-sticker/0.1/test/validator-amp-story-audio-sticker.out

```diff
@@ -0,0 +1,74 @@
+PASS
```

Comment:
could we add another fixture where it fails when its not inside a `amp-story-grid-layer`

Reply:
done.